renpy/display/render.pyx

```diff
@@ -223,6 +223,7 @@ cpdef render(d, object widtho, object heighto, double st, double at):
     xmaximum = style.xmaximum
     ymaximum = style.ymaximum
 
+    # TODO: check this regarding the new absolute.compute
```

Comment:
xmaximum and ymaximum officially only support integers, but this piece of code which seems to be used by all displayables proves otherwise. If there is no incompatibility with another part of the code, I'd advocate upgrading the style-properties doc (which by the way says offset support all position types, and I think it's not actually the case), and implement the new system here.